backend/src/__tests__/backend.test.ts

```diff
@@ -0,0 +1,20 @@
+import app from '../server';
+import supertest from 'supertest';
+import http from 'http';
+
+let server: any;
+let request: any;
```

Comment:
```suggestion
let server: http.Server;
let request: supertest.SuperTest<supertest.Test>;
```
We can give these explicit types